tgui/packages/tgui/interfaces/Vote.js

```diff
@@ -0,0 +1,212 @@
+import { useBackend } from "../backend";
+import {
+  Box,
+  Icon,
+  Stack,
+  Button,
+  Section,
+  NoticeBox,
+  LabeledList,
+  Collapsible,
+} from "../components";
+import { Window } from "../layouts";
+
+export const Vote = (props, context) => {
+  const { data } = useBackend(context);
+  const { mode, question, lower_admin } = data;
+
+  // Adds the voting type to title if there is an ongoing vote
+  let windowTitle = "Vote";
+  if (mode) {
+    windowTitle
+      += ": " + (question || mode).replace(/^\w/, c => c.toUpperCase());
+  }
+
+  return (
+    <Window resizable title={windowTitle} width={400} height={500}>
+      <Window.Content>
+        <Stack fill vertical>
+          {!!lower_admin && (
```

Comment:
`lower_admin` should already be a variable. What's the truthiness cast for?

Reply:
Byond sends numbers instead of bools, and numbers are rendered in JSX.

Reply:
Fair enough.